### ISSUE TITLE
Warning: torch.nn.utils.weight_norm

### ISSUE THREAD

**Issue by mynona** [remsky/Kokoro-FastAPI#247]
08:03:25 AM | INFO     | main:57 | Loading TTS model and voice packs...
08:03:25 AM | INFO     | model_manager:38 | Initializing Kokoro V1 on cpu
08:03:25 AM | DEBUG    | paths:101 | Searching for model in path: /app/api/src/models
08:03:25 AM | INFO     | kokoro_v1:45 | Loading Kokoro model on cpu
08:03:25 AM | INFO     | kokoro_v1:46 | Config path: /app/api/src/models/v1_0/config.json
08:03:25 AM | INFO     | kokoro_v1:47 | Model path: /app/api/src/models/v1_0/kokoro-v1_0.pth
/app/.venv/lib/python3.10/site-packages/torch/nn/modules/rnn.py:123: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.2 and num_layers=1
  warnings.warn(
/app/.venv/lib/python3.10/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
root@raumnebenan:/home/kaeru/swag#

**Comment by fireblade2534** [comment]
This should not really be a problem but I took a look and most likely the code that is causing that is in the kokoro package
